crowbar_framework/app/views/nodes/_buttons_power.html.haml

```diff
@@ -10,15 +13,30 @@
     %ul.dropdown-menu{ role: "menu" }
       - if @node[:platform_family] != 'windows'
         %li{ role: "presentation" }
-          = link_to t(".reboot"), hit_node_path(@node.handle, "reboot"), role: "menuitem", remote: true
+          = link_to t(".reboot"),
+            hit_node_path(@node.handle, "reboot"),
+            role: "menuitem",
+            data: { blockui_click: t(".blockui_reboot") }
         %li{ role: "presentation" }
-          = link_to t(".shutdown"), hit_node_path(@node.handle, "shutdown"), role: "menuitem", remote: true
+          = link_to t(".shutdown"),
+            hit_node_path(@node.handle, "shutdown"),
+            role: "menuitem",
+            data: { blockui_click: t(".blockui_shutdown") }
         %li.divider{ role: "presentation" }
       - if @node.bmc_set?
         %li{ role: "presentation" }
-          = link_to t(".poweron"), hit_node_path(@node.handle, "poweron"), role: "menuitem", remote: true, "data-confirm" => t("are_you_sure")
+          = link_to t(".poweron"),
+            hit_node_path(@node.handle, "poweron"),
+            role: "menuitem",
+            data: { blockui_click: t(".blockui_poweron"), confirm: t("are_you_sure") }
```

Comment:
`confirm: t("are_you_sure")` is not there for reboot/shutdown/powercycle/poweroff.